datasets/BEIS-Renewables-obligation-certificates-and-generation-monthly/main.py

```diff
@@ -0,0 +1,124 @@
+#!/usr/bin/env python
+# coding: utf-8
+# +
+from gssutils import *
+import pandas as pd
+import json
+
+scraper = Scraper(seed="info.json")
+scraper.distributions = [x for x in scraper.distributions if hasattr(x, "mediaType")]
+scraper
+
+#Add cubes class
+cubes = Cubes("info.json")
+
+#Add TransformTrace
+trace = TransformTrace()
+# -
+
+# Extract latest distribution  of the required dataset and datasetTitle
+distribution  = scraper.distribution(latest=True, title = lambda x:"Renewables obligation: certificates and generation (monthly)" in x)
+datasetTitle = distribution.title
+distribution
+
+# Extract all the tabs and its content from the spread sheet 
+tabs = distribution.as_databaker()
+
+# List out all the tab names to cross verify with spread sheet
+for tab in tabs:
+    print(tab.name)
+
+columns = ["Technology Group", "Generation Type", "Roc Per Mwh", "Period", "Qtr", "Month", "Element"]
+
+# +
+# Filtering the tabs which are required and start stage-1 transform
+tabs_i_want = ["Financial Year", "Quarter", "FY-only sites", "Month"]
+
+tabs = [x for x in tabs if x.name in tabs_i_want]
+for tab in tabs:
+    print(tab.name)
+    
```

Comment:
```suggestion
    remove = tab.filter(contains_string("Data are sourced")).expand(RIGHT).expand(DOWN)
```

Reply:
Captured footer so that it can be removed